src/Tizen.Multimedia.Vision/MediaVision/InferenceFaceDetector.cs

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using InteropFD = Interop.MediaVision.InferenceFaceDetection;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Provides the ability to detect faces.
+    /// </summary>
+    /// <feature>http://tizen.org/feature/vision.inference</feature>
+    /// <feature>http://tizen.org/feature/vision.inference.face</feature>
+    /// <since_tizen> 12 </since_tizen>
+    public class InferenceFaceDetector : IDisposable
+    {
+        private IntPtr _handle;
+        private bool _disposed;
+
+        /// <summary>Initializes a new instance of the <see cref="InferenceFaceDetector"/> class.</summary>
+        /// <exception cref="NotSupportedException">The required features are not supported.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public InferenceFaceDetector()
+        {
+            ValidationUtil.ValidateFeatureSupported(VisionFeatures.Inference);
+            ValidationUtil.ValidateFeatureSupported(VisionFeatures.InferenceFace);
+
+            InteropFD.Create(out _handle).Validate("Failed to create inference face detector.");
+
+            try
+            {
+                InteropFD.Configure(_handle).Validate("Failed to configure inference face detector.");
+                InteropFD.Prepare(_handle).Validate("Failed to prepare inference face detector.");
+            }
+            catch (Exception e)
+            {
+                Log.Error(MediaVisionLog.Tag, e.ToString());
+                InteropFD.Destroy(_handle);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Finalizes an instance of the InferenceFaceDetector class.
+        /// </summary>
+        ~InferenceFaceDetector()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Detects faces on the source image synchronously.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="InferenceFaceDetectorResult.BoundingBoxes"/> can be empty, if there's no detected face.
+        /// </remarks>
+        /// <param name="source">The image data to detect faces.</param>
+        /// <returns>The BoundBoxes of detected face.</returns>
```

Comment:
this one of 2 places where it slipped,   BoundBoxes -> BoundingBoxes

Reply:
I'll fix it.